stm32-modules/vacuum-module/firmware/vacuum-pressure-sensor/vacuum_pressure_sensor_hardware.c

```diff
@@ -0,0 +1,64 @@
+#include <stdbool.h>
+
+#include "stm32g4xx_hal.h"
+#include "stm32g4xx_hal_conf.h"
+#include "stm32g4xx_hal_gpio.h"
+#include "stm32g4xx_hal_def.h"
+#include "systemwide.h"
+#include "main.h"
+
+void vacuum_pressure_sensor_hardware_init(void) {
+    GPIO_InitTypeDef GPIO_InitStruct = {0};
+
+    __HAL_RCC_GPIOA_CLK_ENABLE();
+    __HAL_RCC_GPIOB_CLK_ENABLE();
+    __HAL_RCC_GPIOC_CLK_ENABLE();
+
+    // sensor_a_eoc
+    GPIO_InitStruct.Pin = SENSOR_A_EOC_PIN;
```

Comment:
are these names or pin modes switched? the resets are all input and the end-of-conversions are all outputs